draft-pauly-quic-datagram.md

```diff
@@ -179,40 +173,76 @@ Datagram Data:
 
 When an application sends an unreliable datagram over a QUIC connection,
 QUIC will generate a new DATAGRAM frame and send it in the first available
-packet. This frame SHOULD NOT be delayed, but MAY be coalesced
-with other frames.
+packet. This frame SHOULD NOT be delayed by anything other than the
+congestion controller, but MAY be coalesced with other frames.
```

Comment:
I think it could also be delayed by an unvalidated path, I think.

Reply:
Made the text just say we SHOULD send as soon as possible.